cloud-aws/src/main/resources/templates/aws-cf-stack.ftl

```diff
@@ -390,14 +390,17 @@
             <#if defaultInboundSecurityGroup??>
               { "IpProtocol" : "tcp", "FromPort" : "22", "ToPort" : "22", "SourceSecurityGroupId" : "${defaultInboundSecurityGroup}"} ,
               { "IpProtocol" : "tcp", "FromPort" : "${gatewayPort}", "ToPort" : "${gatewayPort}", "SourceSecurityGroupId" : "${defaultInboundSecurityGroup}"},
+              { "IpProtocol" : "tcp", "FromPort" : "${knoxPort}", "ToPort" : "${knoxPort}", "SourceSecurityGroupId" : "${defaultInboundSecurityGroup}"},
             </#if>
             <#if cloudbreakPublicIp??>
               { "IpProtocol" : "tcp", "FromPort" : "22", "ToPort" : "22", "CidrIp" : "${cloudbreakPublicIp}/32"} ,
               { "IpProtocol" : "tcp", "FromPort" : "${gatewayPort}", "ToPort" : "${gatewayPort}", "CidrIp" : "${cloudbreakPublicIp}/32"},
+              { "IpProtocol" : "tcp", "FromPort" : "${knoxPort}", "ToPort" : "${knoxPort}", "CidrIp" : "${cloudbreakPublicIp}/32"},
```

Comment:
I think this always opens the Knox for the public IP. I think we need it only when Knox is enabled and set to CENTRAL.

Gateway.java:

    @Column(nullable = false)
    private Boolean enableGateway;

    @Column(nullable = false)
    @Enumerated(EnumType.STRING)
    private GatewayType gatewayType = GatewayType.INDIVIDUAL;